src/views/PageSearchPortal.vue

```diff
@@ -152,17 +152,33 @@ export default {
                 // This will hamper interested VO section experience,
                 // because interested VO does not change the URL and
                 // reload the page will take to /search-portal
-                onConfirm: this.handleConfirm
+                onConfirm: this.handleConfirm,
             });
         },
         async searchRequestWithMobile(voMobile) {
             if (voMobile != '') {
-                // Update Search Text with voMobile
-                this.updateMobileInput(voMobile);
-                this.$router.push({
-                    path: this.$route.fullPath,
-                    query: { mobile: voMobile },
-                });
+                // Sanitize Mobile Number
+                const sanitizeMobileNumber = this.sanitizeMobile(
+                    voMobile.trim(),
```

Comment:
do we need to trim it. Since the space characters will be anyways removed while sanitizing

Reply:
Yes, we don't need to trim it